eclipsesource/perfetto
# Resize track shells horizontally

#### What it does

Gives the user the ability to resize the column of track shells in the timeline.

#### How to test

1. Hover over to the right of timeline buttons till it changes the cursor (is 4px wide)
2. Drag the line to the right
3. UI should change accordingly

#### Media

https://github.com/eclipsesource/perfetto/assets/121060410/2892fd9f-2088-4367-b5bd-81416e309b15
